README.md

```diff
@@ -217,6 +217,17 @@ The ViewModifiers are used to customise the look and feel of the BottomSheet.
 - Changing the threshold does not affect whether either option is enabled.
 - The default threshold is 30% (0.3).
 
+`.iPadFloatingSheet(Bool)`: Makes the iPad's bottom sheet be an actual bottom sheet when set to true.
+- The default is `false`.
+
+`.relativeSheetWidth(Double?)`: Sets the relative width of the sheet.
```

Comment:
Also not sure if this should be changeable on iPhone.

Reply:
It won't break anything, so why not give devs more control over the sheet width? 😜

Reply:
Mhm because I think that soon people will ask for a modifier/change to only change the the sheet width on Mac, IPad and iPhone landscape mode. I think nobody wants to make the sheet smaller in iPhone portrait mode.

Reply:
Devs can input a different value in the modifier depending on the platform (which they can check programmatically); But if you insist, I can remove the functionality for iPhone?

Reply:
No I think your right. Thanks for your input.